packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -997,6 +999,62 @@ class GraphiQLWithContext extends React.Component<
             </div>
           </div>
         </div>
+        <Dialog
+          isOpen={this.state.showShortKeys}
+          onDismiss={() => {
+            this.setState({ showShortKeys: false });
+          }}>
+          <div className="graphiql-dialog-header">
+            <div className="graphiql-dialog-title">Short Keys</div>
+            <Dialog.Close
+              onClick={() => {
+                this.setState({ showShortKeys: false });
+              }}
+            />
+          </div>
+          <div className="graphiql-dialog-section">
+            <ul className="graphiql-short-keys">
```

Comment:
I'm guessing you plan on moving this to JSON eventually?

Also, you could add a little note on this dialog if you want that contains a little link to these docs, keyed by `prop.keyMap`:
https://codemirror.net/5/demo/sublime.html
https://codemirror.net/5/demo/vim.html

also, just noticed i was completely wrong about that vim PR! this does have insert mode already, i forgot, this should be good to say we already have vim support. i will correct myself there.